shared/js/utils.js

```diff
@@ -36,6 +36,12 @@ require.scopes.utils = ( () => {
         }
     }
 
+    function isChromeBrowser () {
```

Comment:
We're doing this in background.js too. Do you want to generalize that and make it a util function? https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/master/shared/js/background.js#L30

Reply:
Yeah, I thought about that but rejected it and opted for utils.isChromeBrowser because it is synchronous. The other one you point to is async, and the boot up/init for this is tricky. Want sync for this, thus new fn in utils.